client/homescreen/activity-panel/orders/index.js

```diff
@@ -27,254 +26,305 @@ import {
 	ActivityCard,
 	ActivityCardPlaceholder,
 } from '../../../header/activity-panel/activity-card';
-import { CurrencyContext } from '../../../lib/currency-context';
 import './style.scss';
 
-class OrdersPanel extends Component {
-	recordOrderEvent( eventName ) {
-		recordEvent( `activity_panel_orders_${ eventName }`, {} );
-	}
+function recordOrderEvent( eventName ) {
+	recordEvent( `activity_panel_orders_${ eventName }`, {} );
+}
 
-	renderEmptyCard() {
-		return (
-			<Fragment>
-				<ActivityCard
-					className="woocommerce-empty-activity-card"
-					title=""
-					icon=""
```

Comment:
Most of the changes here are converting the component to a functional component and updating the `OrdersQuery` see below.